Examples/Tutorial/KTProcessorTemplate.cc

```diff
@@ -26,9 +26,9 @@ namespace Katydid
             fDummySignal3("dummy-signal-3", this),
```

Comment:
KTProcessorTemplate.hh/cc should not be moved inside the Tutorial directory.  They're meant to be template files that any developer can copy to make a new processor.